closed/autoconf/custom-hook.m4

```diff
@@ -271,6 +272,26 @@ AC_DEFUN([OPENJ9_PLATFORM_EXTRACT_VARS_FROM_CPU],
   esac
 ])
 
+AC_DEFUN([OPENJ9_CONFIGURE_INLINETYPES],
+[
+  AC_MSG_CHECKING([for inlinetypes])
+  AC_ARG_ENABLE([inlinetypes], [AS_HELP_STRING([--enable-inlinetypes], [enable InlineType support @<:@disabled@:>@])])
+  OPENJ9_ENABLE_INLINETYPES=false
+
+  if test "x$enable_inlinetypes" = xyes ; then
+    AC_MSG_RESULT([yes (explicitly enabled)])
+    OPENJ9_ENABLE_INLINETYPES=true
+  elif test "x$enable_inlinetypes" = xno ; then
+    AC_MSG_RESULT([no (explicitly disabled)])
+  elif test "x$enable_inlinetypes" = x ; then
+    AC_MSG_RESULT([no (default)])
+  else
+    AC_MSG_ERROR([--enable-inlinetypes accepts no argument])
+  fi
+  AC_SUBST(OPENJ9_ENABLE_JITSERVER)
```

Comment:
This should say `OPENJ9_ENABLE_INLINETYPES` instead of `OPENJ9_ENABLE_JITSERVER`.